doc/tutorials/acrn-sanity-test.rst

```diff
@@ -0,0 +1,347 @@
+ACRN Sanity Test
+################
+
+The document describes the daily sanity test for ACRN by validation team, which 
+includes a set of system level core tests of basic functionality to demonstrate 
+ACRN hypervisor’s availability. User could manually run these after setup ACRN. 
+These cases support in ``kbl-nuc-i7``.
+
+ACRN Sanity Test Cases
+######################
+
+Hypervisor
+##########
+
+Introduction
+************
+
+ST_HV_Bootloader_LaaG_VUEFI_Default
+***********************************
+
+* Description
+
+The test case focus on testing boot up a LaaG using a virtual UEFI (OVMF) firmware and the default ``launch_uos.sh``.
+
+* Prerequisites
+
+Service OS boots up normally
+
+* Test Steps
+
+1. Launch SOS
+#. Use ``OVMF.fd`` to launch clearlinux using the pre-installed default ``launch_uos.sh`` script
+
+  .. code-block:: none
+
+     $ sudo /usr/share/acrn/samples/nuc/launch_uos.sh
+
+  .. note::
+
+     script will use uos.img in path /home/clear/uos/. if your account name is not clear
+
+     $ mkdir -p /home/clear/uos/
+
+     $ cd /home/clear/uos/
+
+     $ curl https://download.clearlinux.org/releases/xxxxx/clear/clear-xxxxx-kvm.img.xz -o uos.img.xz
+
+     $ unxz uos.img.xz
+
+
+* Expected Result
+
+LaaG boot up normally
```

Comment:
No need to update the kernel and modules (kernel-iot-lts2018) in that image?

This is a departure from the Getting Started Guide - see https://projectacrn.github.io/latest/getting-started/apl-nuc.html#set-up-reference-uos

Reply:
sanity test will use the default kernel. Can boot up successfully.

Reply:
sanity test will use the default kernel. Can boot up successfully.